packages/transport/src/transports/abstract.ts

```diff
@@ -163,6 +165,7 @@ export abstract class AbstractTransport extends TypedEmitter<{
         signal.addEventListener('abort', abort);
 
         this.logger = logger;
+        this.signal = signal;
```

Comment:
because of https://github.com/trezor/trezor-suite/pull/14180/commits/c3d4d2fbff5c4009ba377b99fe1b904fd9c5650f#diff-3a72a34ba3cff1d3318d6902cfe59925e7c3b6e9fc7270295e294d6f9677f18bR32